omero/sysadmins/version-requirements.txt

```diff
@@ -1229,9 +1229,9 @@ Ice
       - from June 2015
       - to TBA
       - Unsupported
-      - Unsupported
+      - Supported
       - Recommended
-      - :zerocforum:`Ice 3.6.0 <announcements/6631-ice-3-6-0-and-ice-touch-3-6-0-released>`
```

Comment:
It's documenting the release history; it's just a reference.  It has absolutely nothing to do with the support status.